config/cesm/machines/config_machines.xml

```diff
@@ -354,29 +354,30 @@ This allows using a different mpirun command to launch unit tests
   </machine>
 
    <machine MACH="casper">
-    <DESC>NCAR GPU platform, os is Linux, 36 pes/node, batch system is slurm</DESC>
+    <DESC>NCAR GPU platform, os is Linux, 36 pes/node, batch system is pbs</DESC>
     <NODENAME_REGEX>casper*</NODENAME_REGEX>
-    <!-- MPT sometimes timesout at model start time, the next two lines cause
-    case_run.py to detect the timeout and retry FORCE_SPARE_NODES times -->
     <OS>LINUX</OS>
-    <COMPILERS>pgi</COMPILERS>
-    <MPILIBS compiler="pgi" >openmpi</MPILIBS>
+    <COMPILERS>pgi,intel,nvhpc,pgi-gpu,nvhpc-gpu</COMPILERS>
+    <MPILIBS compiler="pgi" >openmpi,mpi-serial</MPILIBS>
```

Comment:
mpi-serial is implied - there is no reason to list it here.

Reply:
In fact since your only mpi lib is openmpi you can just have one entry
<MPILIBS>openmpi</MPILIBS>